### DISCUSSION TITLE
[DGS v8.5 Spring GraphQL] response null in DgsContext WebRequest

### DISCUSSION THREAD

**Discussion by ps-tunnelsoft** [Netflix/dgs-framework#1871] [Q&A]
Hello all,

I've recently had the opportunity to test the new graphql-dgs-spring-graphql-starter  from the DGS Framework v8.5. While trying out the new starter, I encountered an issue where the WebRequest in the DgsContext is returning null as response.

I wanted to follow the documentation 

```
@DgsQuery
@DgsMutation
public String updateCookie(@InputArgument String value, DgsDataFetchingEnvironment dfe) {
    DgsWebMvcRequestData requestData = (DgsWebMvcRequestData) dfe.getDgsContext().getRequestData();
    ServletWebRequest webRequest = (ServletWebRequest) requestData.getWebRequest();
    javax.servlet.http.Cookie cookie = new javax.servlet.http.Cookie("mydgscookie", value);
    webRequest.getResponse().addCookie(cookie);

    return value;
}

```
This is not an issue with the normal graphql-dgs-spring-boot-starter. That one works without any issue. 

Does anybody have an idea or experience with the Spring Boot GraphQL Starter? Your input would be greatly appreciated. 
Thanks.

**Comment by srinivasankavitha** [comment]
Another user reported the same and we just merged a fix for this issue here: https://github.com/Netflix/dgs-framework/pull/1870

We will release next week.
